front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -158,4 +158,17 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
       contribution_purpose_descrip: this.contributionPurposeDescrip,
     });
   }
+
+  onContactLookupSelect(value: Contact) {
+    if (value && typeof value.id === 'number') {
```

Comment:
type of value.id === 'number' will not work after we switch to UUID primary keys on all the models/tables

Reply:
Nice catch.  I went ahead and reworked some of the model classes to allow us to check instanceof instead.

Reply:
Nice extension of PrimeNg interface